package.json

```diff
@@ -18,8 +18,8 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "format": "prettier --loglevel error --write './src/**/*.ts'",
```

Comment:
Why is this changing?

Reply:
That is because I am using windows 10 and somehow using single quote is not work (But I know this command works on my MacPro)

```
> prettier --loglevel error --write './src/**/*.ts'

[error] No matching files. Patterns tried: './src/**/*.ts' !**/node_modules/** !./node_modules/** !**/.{git,svn,hg}/** !./.{git,svn,hg}/**
```